demo/Vagrantfile

```diff
@@ -22,15 +24,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.synced_folder ".", "/vagrant", disabled: true
 
-  config.vm.provision "shell", privileged: false, inline: <<SCRIPT
-    ## Uncomment these lines to update flynn before starting
-    # sudo apt-get update -qq
-    # sudo apt-get install -y flynn-host
-    # sudo flynn-host download /etc/flynn/version.json
-
-    sudo start flynn-host
+  config.vm.provider "virtualbox" do |v|
+    v.gui = true
```

Comment:
Yeah, help text output in the box terminal would be really cool.

How about I comment out the gui flag for now and open a separate issue for reinstating it once the flynn box can be updated with help text output?

I'm not sure right off hand how to customize the actual login prompt on Debian/Ubuntu.